src/course-unit/xblock-container-iframe/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRef, FC } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { getConfig } from '@edx/frontend-platform';
+
+import { IFRAME_FEATURE_POLICY } from '../constants';
+import { useIFrameBehavior } from './hooks';
+import messages from './messages';
+
+/**
+ * This offset is necessary to fully display the dropdown actions of the XBlock
+ * in case the XBlock does not have content inside.
+ */
+const IFRAME_BOTTOM_OFFSET = 220;
```

Comment:
[question] Do we need to have a static bottom offset and how it can effect on with using different branding?

Reply:
Most likely yes, but not always. Yesterday we discussed that we want to add this offset programmatically if the user opens the last xblock in the iframe and the xblock height is not sufficient to fully display the body of the dropdown list. This functionality will be added later 💯